### PR TITLE
Add ERC4626 cow burner

### PR DESCRIPTION
**Pull request by jubeira**
# Description

An alternative approach to #1365.
This is not my personal preferred approach from an architecture's standpoint, but it's easier to wire up governance-wise and we might be able to use it in the meantime.

Instead of unwrapping at the sweeper, we can deploy a new cow burner that unwraps before posting the order.
It has a few caveats: it must pull the tokens from the sweeper before creating the order and handle a few edge cases. But other than that it's rather reasonable.

Tests pending.

## Type of change

- [ ] Bug fix 
- [x] New feature 
- [ ] Breaking change 
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Optimization: [ ] gas / [ ] bytecode
- [ ] Documentation or wording changes
- [ ] Other

## Checklist:

- [x] The diff is legible and has no extraneous changes
- [x] Complex code has been commented, including external interfaces
- [ ] Tests have 100% code coverage
- [x] The base branch is either `main`, or there's a description of how to merge

## Issue Resolution

N/A

### CODE REVIEW

File: pkg/standalone-utils/contracts/ERC4626CowSwapFeeBurner.sol [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        // This case is not handled by the internal `_burn` function, but it's valid: we can consider that the token
+        // has already been converted to the correct token, so we just forward the result and finish.
+        if (feeToken == targetToken) {
+            // We apply the slippage check, but not deadline as the order settlement is instant in this case.
+            if (exactFeeTokenAmountIn < minTargetTokenAmountOut) {
+                revert AmountOutBelowMin(targetToken, exactFeeTokenAmountIn, minTargetTokenAmountOut);
+            }
+
+            feeToken.safeTransfer(recipient, exactFeeTokenAmountIn);
+            return;
```
Comment (EndymionJkb) [inline pkg/standalone-utils/contracts/ERC4626CowSwapFeeBurner.sol:93]: Consider if/else vs. early return, since there's only one line after it.

**Review by EndymionJkb — APPROVED** [review]
LGTM! The fee sweeper already does the approvals, so it looks like it should work (and with the deployed one). Just a couple comment nits.

File: pkg/standalone-utils/contracts/ERC4626CowSwapFeeBurner.sol (hunk 1 above)
Comment (jubeira) [reply pkg/standalone-utils/contracts/ERC4626CowSwapFeeBurner.sol]: that's fair
